src/main/java/com/oxywire/oxytowns/events/TownDisbandEvent.java

```diff
@@ -0,0 +1,42 @@
+package com.oxywire.oxytowns.events;
+
+import com.oxywire.oxytowns.entities.impl.town.Town;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.HandlerList;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Called when a town has been disbanded by a player.
+ */
+public class TownDisbandEvent extends TownPlayerEvent implements Cancellable {
```

Comment:
This should also be fired on `com.oxywire.oxytowns.command.commands.admin.town.TownDisbandCommand`